.github/workflows/qa-testbench.yaml

```diff
@@ -24,12 +35,17 @@ jobs:
     timeout-minutes: 45
     env:
       IMAGE: "gcr.io/zeebe-io/zeebe"
-      GENERATION_TEMPLATE: "${{ github.event.inputs.generation }}"
-      BRANCH_NAME: "${{ github.event.inputs.branch }}"
+      GENERATION_TEMPLATE: "${{ inputs.generation || 'Zeebe Snapshot' }}"
```

Comment:
:thinking: Isn't that already defaulted to Zeebe Snapshot?

Reply:
Yeah, I guess this is more of a failsafe in case someone added new event triggers. I added it because it happened to me that I didn't put a default value in the input :sweat_smile: